swaps/swaps_service.go

```diff
@@ -819,6 +819,8 @@ func (svc *swapsService) SwapIn(amount uint64, autoSwap bool) (*SwapResponse, er
 							logger.Logger.WithError(err).WithFields(logrus.Fields{
 								"swapId": swap.Id,
 							}).Error("Could not process refund")
+						} else {
+							swapState = constants.SWAP_STATE_REFUNDED
```

Comment:
This shouldn't be needed actually right? since it's done inside the `RefundSwap` method